src/main/jbake/content/blog/2016/04/28/new-packaging.adoc

```diff
@@ -0,0 +1,129 @@
+= New Hawkular packaging
+Heiko W. Rupp, Stefan Negrea
+2016-4-28
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, manageiq, components
+
+_ [ This blog post is was sent before to the hawkular-dev mailing list ] _
+
+With a growing community and an increasingly large number of sub-projects, it is the perfect time to rethink the
+packaging of Hawkular. This post is a blueprint for changes to come to Hawkular as well as establish some patterns
+for futures decisions.
+
+== Motivation
+
+So far we have created Hawkular "All-in-one" releases that contain everything, Hawkular server components, UI,
+the agent and also embedded versions of Cassandra and KeyCloak.
+
```

Comment:
"It’s certainly the case that for community we will still deliver a fully featured all-in-one package" - is this a valid statement given the shift of the UI to miq? cc @pilhuhn @theute